package.json

```diff
@@ -33,7 +33,8 @@
     "sepia": "^2.0.1",
     "snazzy": "^2.0.1",
     "standard": "^5.3.1",
-    "tape": "^4.2.2"
+    "tape": "^4.2.2",
+    "tmp": "0.0.28"
```

Comment:
FYI, that directory will be automatically deleted after process finish. Is it good behaviour for you?